ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -299,6 +299,13 @@ int mca_pml_ucx_init(int enable_mpi_threads)
         params.thread_mode = UCS_THREAD_MODE_SINGLE;
     }
 
+#if HAVE_DECL_UCP_WORKER_FLAG_IGNORE_REQUEST_LEAK
+    if (!ompi_pml_ucx.request_leak_check) {
```

Comment:
should we check for `ompi_pml_ucx.request_leak_check` ? (not a negation)

Reply:
if we don't want leak check - need to pass a flag to disable it